### ISSUE TITLE
Error running `aqt doc`

### ISSUE THREAD

**Issue by ddalcino** [miurahr/aqtinstall#234]
Sorry, I'm just trying to understand how this code works. I tried running `python -m aqt doc 5.12 linux desktop`, and I got this stack trace:

```
Traceback (most recent call last):
  File "/usr/lib64/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib64/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/dave/PycharmProjects/aqtinstall/aqt/__main__.py", line 27, in <module>
    sys.exit(main())
  File "/home/dave/PycharmProjects/aqtinstall/aqt/__init__.py", line 38, in main
    return cli.run()
  File "/home/dave/PycharmProjects/aqtinstall/aqt/installer.py", line 673, in run
    return args.func(args)
  File "/home/dave/PycharmProjects/aqtinstall/aqt/installer.py", line 406, in run_doc
    self._run_src_doc_examples("doc", args)
  File "/home/dave/PycharmProjects/aqtinstall/aqt/installer.py", line 344, in _run_src_doc_examples
    sevenzip = self._set_sevenzip(args)
  File "/home/dave/PycharmProjects/aqtinstall/aqt/installer.py", line 144, in _set_sevenzip
    subprocess.run(
  File "/usr/lib64/python3.9/subprocess.py", line 505, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib64/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib64/python3.9/subprocess.py", line 1696, in _execute_child
    and os.path.dirname(executable)
  File "/usr/lib64/python3.9/posixpath.py", line 152, in dirname
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not Namespace
```

In `aqt/installer.py`, on line 344, we have this line:

        sevenzip = self._set_sevenzip(args)

Is that meant to be `args.external`, as it is in `Cli.run_install` and `Cli.run_tool`?

**Comment by miurahr** [comment]
this is bug. please change  to   ` sevenzip = self._set_sevenzip(args.external)`

**Comment by miurahr** [comment]
enbug in v1.1.5 at https://github.com/miurahr/aqtinstall/commit/e5a062a150d26d04a67612e01607af077e176ea9

**Comment by miurahr** [comment]
Fix v1.1.6 out.
